src/shared/constants/cssBreakpoints.js

```diff
@@ -0,0 +1,5 @@
+export const CSSBreakpoints = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+};
```

Comment:
It's not clear to me whether we are importing everything to our `index.js` in `constants` folder or we are ok with having this and importing the module where we need it

Reply:
Well for the rest of the constants we export from the `index.js` file in the `constants` folder, and then we import something like:
```javascript
import { CSSBreakpoints } from 'shared/constants';
```

Reply:
Not a blocker for me, it doesn't really make any difference

Reply:
ok, changing it 👍